README.md

```diff
@@ -4,7 +4,16 @@ This is the parser for the [RecordFlux](https://github.com/Componolit/RecordFlux
 
 ## Development
 
-As a prerequisite to build the parser, [GNAT Community Edition](https://www.adacore.com/download) >= 2020 needs to be installed. To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
+As a prerequisite to build the parser, the following dependencies need to be installed:
+
+- [GNAT Community Edition](https://www.adacore.com/download) >= 2020
+- [patchelf](https://github.com/NixOS/patchelf)
+- [GNU MP Arithmetic Library](https://gmplib.org/) + development tools. These are provided as packages for various distributions e.g. libgmp-dev (Debian/Ubuntu), gmp-devel (Fedora) or gmp (Arch)
```

Comment:
I like the new version better. Just some nits:
- `-dev` or `-devel` means that (development) headers are provided with the package, there are usually no "development tools" for a library. I would just remove that "+ development tools".
- The distribution is called Arch Linux, not just Arch.
- A sentence should have a dot at the end.

Reply:
Ok, changed that. The package description of the fedora package calls it development tools...  :)

Reply:
Interesting, naming that development tools seems odd to me, but Debian also calls it developers tools in the short description. Anyway, I think the description is good as it is now.